packages/applications-service-api/__tests__/integration/interestedParty.test.js

```diff
@@ -328,5 +328,21 @@ describe('/api/v1/interested-party', () => {
 				);
 			});
 		});
+
+		describe('request with missing required properties', () => {
+			it('should return 400 error with message', async () => {
+				const response = await request.post('/api/v1/interested-party').send({});
+
+				expect(response.status).toEqual(400);
+				expect(response.body).toEqual({
+					code: 400,
+					errors: [
+						"must have required property 'case_ref'",
```

Comment:
Out of scope of this PR but can we make the errors specific to where the property is missing. It seems it could be missing from `req.query` or `req.params` or `req.body` and it will give the same error message regardless.